Update content on kube-dns to coredns configmap translation (#13781)
kubernetes/website: content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

    consul.local:53 {
        errors
        cache 30
        forward . 10.150.0.1
    }
```
In Kubernetes version 1.10 and later, kubeadm supports automatic translation of the CoreDNS ConfigMap from the kube-dns ConfigMap.

> ***Note: While kube-dns accepts a service name (eg: ns.foo.com) for stubdomain and nameserver, CoreDNS does not support this feature. 

Reviewer: ```suggestion
***Note: While kube-dns accepts a dns name (eg: ns.foo.com) for stubdomain and nameserver, CoreDNS does not support this feature. 
```

> During translation, if a service name is detected, it will be omitted while translating to the equivalent CoreDNS config.***

Reviewer: ```suggestion
During translation, all FQDN nameservers will be be omitted from the CoreDNS config.***
```
